Dockerfile-py2.7

```diff
@@ -25,11 +25,14 @@ ENV DISPLAY :99
 
 WORKDIR /workspace/Qt.py
 ENTRYPOINT cp -r /Qt.py /workspace && \
-    python build_caveats_tests.py && \
+    python build_caveats.py && \
+    python build_membership.py && \
 	Xvfb :99 -screen 0 1024x768x16 2>/dev/null & \
-	sleep 3 && \
+	sleep 10 && \
```

Comment:
Woah, seems like this one has no end.

Let's find a better way of handling this. There must be some way of waiting for a signal/sign of sorts, and continuing once that happens?